### PR TITLE
Reuse handle when bulk-reloading

### PR DESCRIPTION
**Pull request by noctux**
As all feed updates target the same host when an remote API is used, they can benefit from cURL-handle and therefore connection reuse. This is implemented by this commit.

At a cursory glance, it seems to pay off, even though the total payoff is not as siginificant as I've hoped (most of the time seems to be spend waiting for the network, even though the measurements were taken between servers with two symmetric gigabit links. So we save on CPU time, but without ostendable speedups in the actual operation):
```
Old:
for i in $(seq 0 7); do time ./newsboat -x reload; done
./newsboat -x reload  15.29s user 1.32s system 93% cpu 17.766 total
./newsboat -x reload  15.65s user 1.44s system 96% cpu 17.772 total
./newsboat -x reload  15.40s user 1.37s system 59% cpu 27.970 total
./newsboat -x reload  15.64s user 1.39s system 98% cpu 17.305 total
./newsboat -x reload  15.50s user 1.31s system 89% cpu 18.747 total
./newsboat -x reload  15.08s user 1.34s system 87% cpu 18.796 total
./newsboat -x reload  15.45s user 1.24s system 91% cpu 18.328 total
./newsboat -x reload  15.54s user 1.32s system 89% cpu 18.753 total

New:
for i in $(seq 0 7); do time ./newsboat -x reload; done
./newsboat -x reload  8.77s user 1.24s system 73% cpu 13.630 total
./newsboat -x reload  9.42s user 1.39s system 71% cpu 15.130 total
./newsboat -x reload  8.77s user 1.32s system 75% cpu 13.405 total
./newsboat -x reload  8.42s user 1.31s system 68% cpu 14.257 total
./newsboat -x reload  8.49s user 1.37s system 63% cpu 15.611 total
./newsboat -x reload  8.76s user 1.38s system 68% cpu 14.798 total
./newsboat -x reload  8.34s user 1.34s system 55% cpu 17.419 total
./newsboat -x reload  8.61s user 1.33s system 70% cpu 14.000 total
```

### CODE REVIEW

File: include/utils.h [hunk 1]
```diff
@@ -55,7 +55,8 @@ class utils {
 				const std::string& url,
 				configcontainer * cfgcont = nullptr,
 				const std::string& authinfo = "",
-				const std::string* postdata = nullptr);
+				const std::string* postdata = nullptr,
+				CURL* easyhandle = 0);
```
Comment (Minoru) [inline include/utils.h]: We're on C++11 now, so `nullptr` should be preferred to `0`.

**Comment by noctux** [comment]
> s/0/nullptr/ 

Urgs, yeah, of course. I don't know how that got there... fixed now. By the way, did I mention that styleguide thing? :)

**Comment by Minoru** [comment]
Thank you, @noctux!

**Comment by noctux** [comment]
For crossreferencing: #16

**Pull request merged**